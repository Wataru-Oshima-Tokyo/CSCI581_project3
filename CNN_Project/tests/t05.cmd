  input_2C.txt weights_2C.txt  structure_2C.txt 0.01
