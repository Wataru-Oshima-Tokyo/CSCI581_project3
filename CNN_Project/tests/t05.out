0.5378101854290160 0.5403463667307705 0.5417393574273973 0.5371599872127125 
